ray-operator/config/rbac/kustomization.yaml

```diff
@@ -5,6 +5,10 @@ resources:
 - leader_election_role_binding.yaml
 - service_account.yaml
 
-commonLabels:
-  app.kubernetes.io/name: kuberay
-  app.kubernetes.io/component: kuberay-operator
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Is that change really needed?

Reply:
These were changes made due to a Warning I received during the kustomize deployments regarding deprecations, I followed the instruction to clean up any other discrepancies. However this could be addressed separately:
```
# Warning: 'bases' is deprecated. Please use 'resources' instead. Run 'kustomize edit fix' to update your Kustomization automatically.
```

Reply:
Right, ideally these changes should be contributed to KubeRay upstream, and rebase our fork on top. We want to keep the patches as small as possible in the later.

Reply:
Perfect, thanks @astefanutti :)